src/Control/Carrier/Error/Either.hs

```diff
@@ -1,4 +1,7 @@
 {-# LANGUAGE FlexibleInstances, GeneralizedNewtypeDeriving, MultiParamTypeClasses, TypeOperators, UndecidableInstances #-}
+
+{- | A carrier for an 'Error' effect.
+-}
```

Comment:
I was mostly suggesting `{-…-}` for when we have several lines. For single line comments, I think it’s just adding noise.